Cargo.toml

```
@@ -7,11 +7,16 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0"
-log = "0.4"
+log = "0.4.18"
 serde = "1.0.159"
 serde_json = "1.0.72"
 cln-plugin = { git = "https://github.com/chrisguida/lightning", version = "0.1.2", branch = "feat/cln-plugin-send-notifs" }
-bdk = "0.27.1"
+# cln-plugin = { path = "../../lightning/plugins" }
+# bdk = "0.27.1"
+# bdk = { version = "0.28", default-features=false, features = ["std", "key-value-db","async-interface", "use-esplora-async"] }
+bdk = { git = "https://github.com/bitcoindevkit/bdk", version = "1.0.0-alpha.0" }
```

We just released alpha.1 and is much closer to final 1.0.0 API.

```suggestion
bdk = { git = "https://github.com/bitcoindevkit/bdk", version = "1.0.0-alpha.1" }
```